### ISSUE TITLE
[QInput] When in `autogrow` mode, ancestor container unexpectedly scrolls to top when input

### ISSUE THREAD

**Issue by xuzuodong** [quasarframework/quasar#15023]
### What happened?

If `QInput` component is in `autogrow` mode and is placed in a scrollable container, when scrolling down container, and then change content in `QInput`, the container unexpectedly scrolls to top. 

![屏幕录制2022-12-01 17 26 42](https://user-images.githubusercontent.com/51878637/205016303-eac5516a-43f1-4076-98ed-434841e92217.gif)

### What did you expect to happen?

the container won't scroll when inputting

### Reproduction URL

https://stackblitz.com/edit/quasarframework-wjsmsl?file=src/App.vue

### How to reproduce?

1. Go to the repro link
2. In live preview area, enter several lines to fill the `QInput` component , until length of the `QInput` component big enough so that the parent `div` element becomes scrollable
3. scroll to bottom
4. type something in `QInput`

### Flavour

Vite Plugin (@quasar/vite-plugin)

### Areas

Components (quasar)

### Platforms/Browsers

Chrome, Safari

### Quasar info output

```Shell
Operating System   	Darwin(22.1.0) - darwin/arm64
NodeJs             	16.15.0

Global packages
  NPM              	8.5.5
  yarn             	1.22.11
  @quasar/cli      	1.2.0
  @quasar/icongenie	Not installed
  cordova          	Not installed

Networking
  Host             	Porsche.local
  en0              	172.16.88.101
```

**Comment by Zaphod-Beeblebrox** [comment]
I've run into this as well.  I solved it by turning off  auto-grow and dynamically computing the rows Prop:
`<q-input
          type="textarea"

          clearable

          :rows="notesRows"

          label="Notes"

          stack-label

          v-model="scopeNotes"

        >

const notesRows = computed(() => {

  let result = 5;

  const usedrows = scopeNotes.value ? scopeNotes.value.split(`\n`).length : 0;

  result = Math.min(15, Math.max(5, usedrows));

  return result;

});
`
In my case I allow a dynamic range between 5 and 15 lines.  adjust to your liking

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v2.10.3 and v1.22.3
